src/silvimetric/resources/metric.py

```diff
@@ -86,11 +88,12 @@ def m_max(data):
 def m_stddev(data):
     return np.std(data)
 
+#TODO change to correct dtype
 Metrics = {
-    'mean' : Metric('mean', np.float64, m_mean),
-    'mode' : Metric('mode', np.float64, m_mode),
-    'median' : Metric('median', np.float64, m_median),
-    'min' : Metric('min', np.float64, m_min),
-    'max' : Metric('max', np.float64, m_max),
-    'stddev' : Metric('stddev', np.float64, m_stddev),
+    'mean' : Metric('mean', np.float32, m_mean),
```

Comment:
maybe the type is defaulted to np.float32. It's very rare that users would make any other type.